README.md

```diff
@@ -1,2 +1,96 @@
 # opentok-session-lambda-python
-Python app, including Lambda deployment, to provide OpenTok session upon request.
+A Flask based Python Lambda function to provide OpenTok session upon request.
+
+## Prerequisites
+* Python
```

Comment:
Since the `serverless.yml` file requires v3.7, do you want to specify that here explicitly? It would resolve the issue I had with dependencies not loading because I was using v3.8.

Reply:
+1

Reply:
Added disclaimer in later commit